Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="15.0">
   <PropertyGroup>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <MSBuildTreatWarningsAsErrors>true</MSBuildTreatWarningsAsErrors>
+    <MSBuildTreatWarningsAsErrors>false</MSBuildTreatWarningsAsErrors>
```

Comment:
disabled this for now, as there are warnings about .NET 5 being unsupported

Reply:
but isn't this PR upgrading from v5 to v6? then this warning should be gone in this PR